docs/source/qpu.rst

```diff
@@ -9,7 +9,9 @@ them using pyQuil.
 .. note::
 
     In order to run experiments on the QPU you will need a specially upgraded API key.  If you are
-    interested in running these experiments, then email us at support@rigetti.com
+    interested in running these experiments, then email us at support@rigetti.com.
+
+    You will also need to use the following URL endpoint: https://job.rigetti.com/beta
```

Comment:
Ohh...@chris and @jeff told me I had to change the URL in the pyquil_config ini file, and that wasn't reflected in the instructions. Is that not correct?